src/components/slider/slider.ts

```diff
@@ -215,6 +228,80 @@ export class MdSlider implements AfterContentInit {
   }
 
   /**
+   * Calls the appropriate method for calculating the tick separation depending on what the tick
+   * interval is. If there is no tick interval or the interval is set to something other than a
+   * number or 'auto', nothing happens.
+   */
+  private _calculateTickSeparation() {
```

Comment:
Perhaps call this `_updateTickSeparation`? When I see `calculate`, I think it should return something.